src/planner.tsx

```diff
@@ -145,7 +145,10 @@ export function planEnsurePrivateContact(
 
 export function planUpsertRelations(plan: Plan, relations: Relations): Plan {
   const userDB = plan.knowledgeDBs.get(plan.user.publicKey, newDB());
-  const updatedRelations = userDB.relations.set(relations.id, relations);
```

Comment:
so this was the bug you spoke about?

Reply:
no, that bug will be fixed in the next PR